manifests/quick-start/base/httpbin/httpbin-deploy.yaml

```diff
@@ -16,7 +16,12 @@ spec:
       automountServiceAccountToken: false
       containers:
         - name: main
-          image: kennethreitz/httpbin
+          image: mccutchen/go-httpbin
```

Comment:
Are there multiple alternatives? If so, what's the reason to pick this one?

Reply:
I saw this one has good maintenance. But the test is failing due to a different response on oaurth2. I'm considering finding other alternatives or rebuilding this docker image.

Reply:
@terrytangyuan help me review again. Use a new image kong/httpbin that support multiple arch
I checked 2 failing test, they do not relate with my change

Reply:
@terrytangyuan just run test on local for (2) and passed that test. nodejs v19 link for (1) also available now. Plese help me retrigger github action for test again